app/models/search_builder.rb

```diff
@@ -35,7 +35,7 @@ def readable_states
 
   # This is a blacklist of models that should not be presented in search results
   def models_to_solr_clause
-    [FileSet, EphemeraProject, EphemeraVocabulary, EphemeraTerm, FileMetadata, EphemeraField, Template].join(",")
+    [FileSet, EphemeraProject, EphemeraVocabulary, EphemeraTerm, FileMetadata, EphemeraField, Template, Playlist, ProxyFile].join(",")
```

Comment:
Do we want to suppress discovery of Playlists? I thought we wanted to make them private by default, to prevent end users from discovering them, but still allow staff to discover them.